### PR TITLE
At/cb 134 implement inubekit on login pages

### PR DESCRIPTION
**Pull request by Angelbt12**
Implement inubekit on login pages
image
Adjust the Login Pages to change the design-system components to design-system-web (inubekit) components

### CODE REVIEW

File: src/pages/login/outlets/CheckingCredentials/interface.tsx [hunk 1]
```diff
@@ -12,7 +14,7 @@ function CheckingCredentialsUI() {
         </Text>
       </Stack>
       <Stack alignItems="center" direction="column">
-        <Spinner size="large" />
+        <Spinner size="large" appearance="primary" transparent={false} />;
       </Stack>
```
Comment (diegoAndres01) [inline src/pages/login/outlets/CheckingCredentials/interface.tsx:18]: This semicolon should be removed

**Review by diegoAndres01 — CHANGES_REQUESTED** [review]

File: src/pages/login/outlets/CheckingCredentials/interface.tsx (hunk 1 above)
Comment (Angelbt12) [reply src/pages/login/outlets/CheckingCredentials/interface.tsx:18]: ok

**Review by diegoAndres01 — APPROVED** [review]

File: src/pages/login/outlets/Clients/styles.ts [hunk 2]
```diff
@@ -23,8 +19,8 @@ const StyledClients = styled.div`
   }
 
   & button {
-    margin-top: ${({ theme }: IStyledClientsUI) =>
-      `${theme?.spacing?.s300 || inube.spacing.s300}`};
+    margin-top: ${({ theme }) =>
+      `${theme? "24px" : "24px"}`};
```
Comment (carin91) [inline src/pages/login/outlets/Clients/styles.ts:23]: Delete the theme validation and pass directly the 24px

File: src/pages/login/outlets/Clients/styles.ts [hunk 3]
```diff
@@ -42,9 +38,9 @@ const StyledClientsList = styled.div<StyledClientsListProps>`
 `;
 
 const StyledNoResults = styled.div`
-  margin: ${({ theme }: IStyledClientsUI) =>
-    `${theme?.spacing?.s200 || inube.spacing.s200} ${
-      theme?.spacing?.s0 || inube.spacing.s0
+  margin: ${({ theme }) =>
+    `${theme? "16px" : "16px"} ${
```
Comment (carin91) [inline src/pages/login/outlets/Clients/styles.ts:42]: Delete the theme validation and pass directly the 16px

File: src/pages/login/outlets/Clients/styles.ts [hunk 4]
```diff
@@ -42,9 +38,9 @@ const StyledClientsList = styled.div<StyledClientsListProps>`
...
+      theme? "0px" : "0px"
```
Comment (carin91) [inline src/pages/login/outlets/Clients/styles.ts:43]: Delete the theme validation and pass directly the 0px

**Review by carin91 — APPROVED** [review]

File: src/pages/login/outlets/Clients/styles.ts (hunk 2 above)
Comment (Angelbt12) [reply src/pages/login/outlets/Clients/styles.ts:23]: ok

File: src/pages/login/outlets/Clients/styles.ts (hunk 3 above)
Comment (Angelbt12) [reply src/pages/login/outlets/Clients/styles.ts:42]: ok

File: src/pages/login/outlets/Clients/styles.ts (hunk 4 above)
Comment (Angelbt12) [reply src/pages/login/outlets/Clients/styles.ts]: ok

**Pull request merged**